README.rst
=====================
TypoScript Reference
=====================


:Repository:  https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-Typoscript
:Read online: https://docs.typo3.org/typo3cms/TyposcriptReference/

Additional information
======================

* `Contribute to TYPO3 documentation <https://docs.typo3.org/typo3cms/HowToDocument/WritingDocsOfficial/Index.html>`__
* `Contribute to TYPO3 core <https://docs.typo3.org/typo3cms/ContributionWorkflowGuide/>`__
* `Help & Support for TYPO3 <https://typo3.org/help>`__


doc_core_tsref/ChangeLog
2012-04-23  Christopher  <chrissitopher@gmx.de>

	* Fixed task #35279: Prepare release of TSref 4.7.0
	* Fixed task #35279: Regenerated the PDF file
	* Released version 4.7.0 to TER

2012-04-22  Christopher  <chrissitopher@gmx.de>

	* Fixed task #36416: Add note on deprecation of the cObject HTML
	* Fixed task #36422: Add note on removal of value "html_5" for config.doctype

2012-04-20  Martin Holtz  <typo3@martinholtz.de>

	* Fixed bug #36375: added property .GMT to stdWrap.date

2012-04-16  Christopher  <chrissitopher@gmx.de>

	* Fixed task #35445: Add information about in which major version new features were added

2012-03-30  Francois Suter  <francois.suter@typo3.org>

	* Fixed bug #35397: TSref must use Share font for headings

2012-03-26  Christopher  <chrissitopher@gmx.de>

	* Fixed task #31938: Add META.<key>.httpEquivalent property
	* Fixed task #33724: Add conditions as page.includeCSS.if and page.includeJS*.if
	* Fixed task #32662: New config.stat_* options to allow anonymized storage of statistic log information
	* Fixed task #33715: Document orderedStdWrap
	* Fixed task #34307: Add documentation for stdWrap function "cache"
	* Fixed task #35279: Prepare release of TSref 4.7.0

2012-03-25  Christopher  <chrissitopher@gmx.de>

	* Fixed bug #34472: Rule 3 of the optionSplit description is incorrect (thanks to Jochen Weiland and Steffen Ritter)
	* Fixed task #32671: Add stdWrap for HMENU: maxItems, minItems & begin
	* Fixed task #32347: Add documentation for new sub-properties of filelink.icon
	* Fixed task #32320: Add documentation of config.pageTitleSeparator
	* Fixed task #31768: New property config.htmlTag_stdWrap
	* Fixed task #34297: Update references to $TYPO3_CONF_VARS['BE']['forceCharset']

2011-12-09  Martin Holtz  <typo3@martinholtz.de>

	* Added description of #32414: added .GMT as property of strftime

2011-12-01  Christopher  <chrissitopher@gmx.de>

	* Added description of #27694: Speedup backend and allow compression in frontend
	* Fixed task #32227: Prepare release of TSref 4.6.0
	* Fixed task #32227: Regenerated the PDF file
	* Released version 4.6.0 to TER

2011-11-30  Christopher  <chrissitopher@gmx.de>

	* Added description of #17634: Remove default CSS
	* Added description of #20473: Starttime/endtime is not taken into account when caching
	* Added description of #28095: Add hashing algorithms to stdWrap
	* Added description of #28027: Add rounding functions to stdWrap
	* Added description of #19347: Integrate string replacement to stdWrap
	* Added description of #16180: Add random functionality to listNum

2011-11-28  Christopher  <chrissitopher@gmx.de>

	* Fixed some wording in the new texts, fixed cover page and TOC
	* Fixed task #29472: Add notes on removal of properties (alttext and special.userdefined)
	* Fixed task #30443: Add a note on the new FORM object
	* Fixed bug #31782: Fix typo in description of optionSplit for HMENU.(mObj).TMENU
	* Added description of #28032: Add stdWrap for IMAGE.file.params
	* Added description of #17881: Enable stdWrap for select.where
	* Added description of #18822: Add stdWrap to each of the properties of TypoScript "select"
	* Added description of #28018: Add ucfirst and lcfirst to stdWrap
	* Added description of #17896: Capitalizing with stdWrap

2011-11-24  Martin Holtz  <typo3@martinholtz.de>

	* Added documentation for the cObjects MEDIA, SWFOBJECT, QTOBJECT 

2011-09-05  Christopher  <chrissitopher@gmx.de>

	* Fixed bug #28755: List of allowed tables in the cObject CONTENT is incomplete
	* Released version 4.5.1 to TER

2011-06-13  Christopher  <chrissitopher@gmx.de>

	* Fixed bug #27341: Condition Reference should explain general Condition syntax
	* Fixed bug #26872: Remove references to PHP_SCRIPT* cObjects
	* Fixed bug #26537: filelink: Property icon_thumbSize is not documented
	* Fixed bug #26507: Each cObj should have a short introduction of what it does

2011-06-03  Christopher  <chrissitopher@gmx.de>

	* Added description of #26483: Add stdWrap for excludeUidList for HMENU

2011-05-16  Francois Suter  <francois.suter@typo3.org>

	* Follow-up to #26543: Reinstated PHP_SCRIPT* objects description into Appendix C
	* Fixed bug #22220: Removed erroneous mention of ADMPANEL properties
	* Released version 4.5.0 to TER

2011-05-15  Francois Suter  <francois.suter@typo3.org>

	* Fixed bug #26543: Removed description of PHP_SCRIPT* objects
	* Fixed bug #10885: Changed "filename" to "file name" and similar issues

2011-05-11  Francois Suter  <francois.suter@typo3.org>

	* Fixed bug #26522: Document "printBeforeContent" for cObject EDITPANEL

2011-05-05  Christopher  <chrissitopher@gmx.de>

	* Fixed bug #13697: Update screenshots for TYPO3 4.5
	* Fixed bug #22493: Update description of page.inlineJS (which has been removed)
	* Fixed bug #13698: Create a manual version for TYPO3 4.5

2011-05-01  Christopher  <chrissitopher@gmx.de>

	* Added description of #24089: Optimize stdWrap usage for tslib_content
	* Added description of #23677: Add SVG support for all browsers by adding a SVG content element
	* Added description of #24065: Optimize stdWrap usage for TypoScript content element SVG
	* Fixed bug #26541: Improve description of cObject TEMPLATE
	* Added description of #23744: A new TypoScript cObject: FLUIDTEMPLATE
	* Added description of #23897: Optimize stdWrap usage for TypoScript content element FLUIDTEMPLATE

2011-04-28  Christopher  <chrissitopher@gmx.de>

	* Added description of #23528: Allow easy use of lightbox style image enlargement in the frontend
	* Added description of #23533: Implement conditions for current page
	* Added description of #25317: Do not include the "type" parameter in the linkVars list
	* Added description of #23760: Let typolink honour secure filelink configuration
	* Added description of #23921: Optimize stdWrap usage for TypoScript content element HTML

doc_core_tsref_ru/ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext "doc_core_tsref".
#
# Auto generated 16-05-2011 22:05
#
# Manual updates:
# Only the data in the array - everything else is removed by next
# writing. "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'Russian translation of TSref',
	'description' => 'TypoScript Reference - translated into Russian. TSref - is a true reference describing the core Content Objects and functions available for Template building using the TypoScript template engine.',
	'category' => 'doc',
	'shy' => 0,
	'version' => '4.5.0',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'loadOrder' => '',
	'module' => '',
	'state' => 'stable',
	'uploadfolder' => 0,
	'createDirs' => '',
	'modify_tables' => '',
	'clearcacheonload' => 0,
	'lockType' => '',
	'author' => 'Official Documentation',
	'author_email' => 'documentation@typo3.org',
	'author_company' => '',
	'CGLcompliance' => '',
	'CGLcompliance_note' => '',
	'constraints' => array(
		'depends' => array(
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:5:{s:9:"ChangeLog";s:4:"6dfd";s:12:"ext_icon.gif";s:4:"7277";s:22:"doc/doc_core_tsref.pdf";s:4:"ccd3";s:22:"doc/doc_core_tsref.txt";s:4:"8c01";s:14:"doc/manual.sxw";s:4:"b666";}',
	'suggests' => array(
	),
);

?>